mappings/package_F22/transformation/mappings/s1_contracting_authority.rml.ttl

```diff
@@ -360,6 +368,18 @@ tedm:BuyerProfile a rr:TripleMap ;
         ] .
 
 tedm:Organisation a rr:TripleMap ;
+    rr:predicateObjectMap
+        [
+            rr:predicate epo:hasDeliveryGateway ;
+            rr:objectMap
+                [
+                    rr:parentTriplesMap tedm:OrganisationChannel;
+                    rr:joinCondition [
+                        rr:child "if(not(exists(/TED_EXPORT/FORM_SECTION/F03_2014/CONTRACTING_BODY/ADDRESS_CONTRACTING_BODY_ADDITIONAL[1]))) then unparsed-text('https://digest-api.ted-data.eu/api/v1/hashing/fn/uuid/' || encode-for-uri(path()) || '?response_type=raw') else null";
```

Comment:
This should not refer to `FORM_SECTION/F03_2014`, as it is part of the generic Section 1 module, instead it should be defined as a relative path.